src/modules/background-music/dto/background-music-admin.dto.ts

```diff
@@ -0,0 +1,32 @@
+import { ApiProperty } from '@nestjs/swagger';
+
+import { BaseResponseDto } from '@common/base/base.dto';
+
+export class BackgroundMusicAdminDto extends BaseResponseDto {
+  @ApiProperty()
+  name: string;
+
+  @ApiProperty()
+  originalFileName: string;
+
+  @ApiProperty()
+  durationInSeconds: number;
+
+  @ApiProperty()
+  backgroundMusicFileName: string;
+
+  @ApiProperty()
+  backgroundMusicUrl: string;
+
+  @ApiProperty()
+  extension: string;
+
+  @ApiProperty()
+  contentType: string;
+
+  @ApiProperty()
+  contentLength: number;
+
+  @ApiProperty({ nullable: true })
```

Comment:
수정 부탁드립니다
https://quizz-game-io.slack.com/archives/C09BYLHRCTB/p1764057713585319
```suggestion
  @ApiProperty({ type: String ,nullable: true })
```

Reply:
수정했습니다!